### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by nsa-yoda**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,18 @@ A script to customize and secure Tails OS with privacy enhancements and UI tweak
 
 ![](https://raw.githubusercontent.com/Tetrax-10/tails-tweaks/main/assets/showcase.jpg)
 
+## Features
+
+- **Dark Mode**: Switches the desktop and terminal to dark mode.
```
Comment (Tetrax-10) [inline README.md]:
```suggestion
- **Dark Mode**: Sets the system theme to dark mode.
```

File: README.md [hunk 2]
```diff
@@ -4,6 +4,18 @@ A script to customize and secure Tails OS with privacy enhancements and UI tweak
...
+- **Wallpaper**: Changes the desktop wallpaper to a specified image.
+- **Auto Sleep**: Disables automatic sleep mode.
+- **Screen Dim & Lock**: Prevents screen dimming and automatic locking.
```
Comment (Tetrax-10) [inline README.md:12]: separate them, like I said in the other PR.

File: README.md [hunk 3]
```diff
@@ -13,6 +25,40 @@ A script to customize and secure Tails OS with privacy enhancements and UI tweak
...
+#### Paranoid Installation
+
+This method is more secure because it does not require you to clone the repository or download the image files. 
+However, you will need to manually create the `wallpapers` folder and place an image file of your choice in it.
+
+1. Open configure.sh in this GitHub repository in the browser and copy the entire contents.
+2. Paste the contents into a new file called `configure.sh` in the `Persistent` folder.
+3. In the Terminal, run `chmod +x configure.sh` to make the script executable.
+4. Modify the script to your preferences according to the usage instructions below.
+5. Run `configure.sh` to apply the changes.
```
Comment (Tetrax-10) [inline README.md]:
```suggestion
**Note**: Downloading files from the internet is not a wise option when privacy and security are your top priorities. If you really want to be secure, do not trust files from the internet; they may contain stegomalware (hidden malicious code inside a file). I recommend not trusting files from random internet users (including me). For installation I recommend you to manually create the `configure.sh` file by copying its contents from GitHub, and download your wallpaper from trusted sites. Make sure to use [VirusTotal](https://www.virustotal.com/gui/) to check that it is not infected.
```

File: README.md [hunk 4]
```diff
@@ -13,6 +25,40 @@ A script to customize and secure Tails OS with privacy enhancements and UI tweak
...
+
+## Usage
+
+1. **Set Variables**: Edit the script to set your preferences by modifying the variables at the top. Each setting can be enabled or disabled by setting the corresponding variable to `true` or `false`.
+
+   ```bash
+   dark_mode=true
+   dark_terminal=true
```
Comment (Tetrax-10) [inline README.md:36]: merge `dark_terminal` with `dark_mode`.

File: README.md [hunk 5]
```diff
@@ -13,6 +25,40 @@ A script to customize and secure Tails OS with privacy enhancements and UI tweak
...
+   change_wallpaper=true
+   wallpaper_path=/home/amnesia/Persistent/wallpapers/wallpaper.jpg
+   auto_sleep=true
+   dim_screen=true
```
Comment (Tetrax-10) [inline README.md:40]: add `auto_screen_lock`

File: README.md [hunk 6]
```diff
@@ -13,6 +25,40 @@ A script to customize and secure Tails OS with privacy enhancements and UI tweak
...
+   disable_camera=true
+   disable_microphone=true
+   disable_location=true
+   disable_usb_auto_mount=false
```
Comment (Tetrax-10) [inline README.md:44]: set `disable_usb_auto_mount` to true

File: README.md [hunk 7]
```diff
@@ -21,7 +67,19 @@ Everytime you boot into Tails, do step 3.
...
+    <td>
+      <img alt="After" src="https://raw.githubusercontent.com/Tetrax-10/tails-tweaks/main/assets/after.jpg" style="width: 400px;">
+    </td>
   </tr>
 </table>
+
+## License
+
+This script is open-source and available under the [MIT License](LICENSE). Feel free to modify and share it.
+
```
Comment (Tetrax-10) [inline README.md]:
```suggestion
```

File: README.md (hunk 3 above)
Comment (Tetrax-10) [reply README.md]: You can add "Modify the script to your preferences according to the usage instructions below." step to #Installation

**Review by Tetrax-10 — CHANGES_REQUESTED** [review]
Again thx for the PR, you could have made it as a single PR so it can be easily traceable. If possible close this PR and commit the request changes to `easier-config` branch.

**Comment by nsa-yoda** [comment]
> Again thx for the PR, you could have made it as a single PR so it can be easily traceable. If possible close this PR and commit the request changes to `easier-config` branch.

Thanks. Will apply your suggested changes, copy the updated readme to the other PR, and close this one as requested.

**Pull request closed without merging**